app/models/models.scala

```diff
@@ -0,0 +1,30 @@
+package models
+
+import org.joda.time.DateTime
+
+case class Author(name: String,
+  url: String,
```

Comment:
Maybe not everyone wants to link a Twitter account? I was thinking it would be nice to give people the freedom to choose this (Twitter, personal website etc).